root/artist/header.tt

```diff
@@ -1,10 +1,11 @@
-<div class="artistheader">
+[% header = 'artistheader ' _ (html_escape(artist.l_type_name) or l('Artist')) _ 'ico' %]
```

Comment:
Remove `or`…`)` as `artistico` becomes unnecessary (see comment to other file).
Do not forget to remove the undue localization prefix `l_` (see previous review).
Possibly move the `artistheader ` class below since it is already lowercase.